### ISSUE TITLE
Need help with configuring 11ty-garden with an existing 11ty-ghost deployment.

### ISSUE THREAD

**Issue by ronykris** [binyamin/eleventy-garden#37]
**Describe the bug**
I need help to configure 11ty-garden within an existing 11ty-ghost deployment.

**To Reproduce**
Nothing to reproduce as this is not a bug. But I do not know how to reach you.

**Expected behavior**
- I have an eleventy site that's fed with content from my ghost deployment.
- I am looking for an integration with this where my base url has something like Notes (similar to About, Home, etc). on the main page
- The md files (with backlinks) should show up here

**Screenshots**

**Environment (please complete the following information):**
Nothing to enter here.

**Additional context**
Add any other context about the problem here. If there's a GitHub repo and/or live demo of the problem, please include link(s).

**Comment by binyamin** [comment]
I found a couple links which might help you.
- official ghost docs for 11ty (https://ghost.org/docs/api/v2/eleventy/
- official ghost starter for 11ty (https://github.com/TryGhost/eleventy-starter-ghost)
- random tutorial (https://david.darn.es/tutorial/2019/06/01/use-eleventy-to-generate-a-ghost-blog/)

You said that you already have an 11ty-ghost deployment. Are you already getting some content from ghost? How are the notes different?

**Comment by binyamin** [comment]
"But I do not know how to reach you." good point. I'll put a link somewhere. How about the wiki?

**Comment by ronykris** [comment]
I have a text sent out to you on twitter DM. So, I already have eleventy-ghost configured and getting content from Ghost. I now want to my md files with backlinks to show up under "notes" in eleventy.

**Comment by binyamin** [comment]
Got it. Did you check out the links I sent?

**Comment by ronykris** [comment]
I need the digital garden integrated to my existing 11ty site and for that I was hoping to use your template.

**Comment by ronykris** [comment]
official ghost starter for 11ty (https://github.com/TryGhost/eleventy-starter-ghost) This was what i followed to get 11ty-ghost configured. It does have info on integrating the 11ty-garden

**Comment by ronykris** [comment]
All I need is any thing that will read a markdown folder and parse the [[ links.

**Comment by binyamin** [comment]
@ronykris Maybe you missed what I said before. Did you look at the links I sent you? What questions did they _not_ answer? (https://github.com/binyamin/eleventy-garden/issues/37#issuecomment-724752184)

**Comment by Sholanki** [comment]
@binyamin I was trying out the same thing. I went through the 3 links posted by you. I am trying to integrate digital garden by using your repo. The integration is successful, but I am unable to do the backlinks integration. Any help will be appreciated.

I have created 'notes' folder inside 'src' folder and 'notes.11tydata.js' inside notes folder where the backlinks functionality is present. I have put the backlinks.html file first in 'includes' folder, later in '_includes' folder and then in 'partials' folder. It did not seem to work
